shared/utils/currencyConversionPairs.ts

```diff
@@ -0,0 +1,1205 @@
+      input: '0x17b4158805772ced11225e77339f90beb5aae968',
+      outputs: ['0x775eb53d00dd0acd3ec1696472105d579b9b386b'],
+    },
+    {
+      input: '0x370de27fdb7d1ff1e1baa7d11c5820a324cf623c',
+      outputs: ['0x775eb53d00dd0acd3ec1696472105d579b9b386b'],
+    },
+    {
+      input: '0x6b175474e89094c44da98b954eedeac495271d0f',
+      outputs: ['0x775eb53d00dd0acd3ec1696472105d579b9b386b'],
+    },
+    {
+      input: '0x775eb53d00dd0acd3ec1696472105d579b9b386b',
+      outputs: [
+        '0xbb3546497a53cd710beb11b84c5240327f145bcb',
+        '0x370de27fdb7d1ff1e1baa7d11c5820a324cf623c',
+        '0xf046b3ca5ae2879c6bacc4d42faf363ee8379f78',
+        '0xf5af88e117747e87fc5929f2ff87221b1447652e',
+        '0x6b175474e89094c44da98b954eedeac495271d0f',
+        '0x17b4158805772ced11225e77339f90beb5aae968',
+        '0x013f29832cd6525c4c6df81c2aae8032a1ff2db2',
+        '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48',
+      ],
+    },
+    {
+      input: '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48',
+      outputs: ['0x775eb53d00dd0acd3ec1696472105d579b9b386b'],
+    },
+    {
+      input: '0xbb3546497a53cd710beb11b84c5240327f145bcb',
+      outputs: ['0x775eb53d00dd0acd3ec1696472105d579b9b386b'],
+    },
+    {
+      input: '0xf046b3ca5ae2879c6bacc4d42faf363ee8379f78',
+      outputs: ['0x775eb53d00dd0acd3ec1696472105d579b9b386b'],
+    },
+    {
+      input: '0xf5af88e117747e87fc5929f2ff87221b1447652e',
+      outputs: ['0x775eb53d00dd0acd3ec1696472105d579b9b386b'],
+    },
+  ],
+  xdai: [
+    {
+      input: '0x17b4158805772ced11225e77339f90beb5aae968',
+      outputs: ['0x775eb53d00dd0acd3ec1696472105d579b9b386b'],
+    },
+    {
+      input: '0x5fd896d248fbfa54d26855c267859eb1b4daee72',
+      outputs: ['0x775eb53d00dd0acd3ec1696472105d579b9b386b'],
+    },
+    {
+      input: '0x775eb53d00dd0acd3ec1696472105d579b9b386b',
+      outputs: [
+        '0xddafbb505ad214d7b80b1f830fccc89b60fb7a83',
+        '0x5fd896d248fbfa54d26855c267859eb1b4daee72',
+        '0xfadc59d012ba3c110b08a15b7755a5cb7cbe77d7',
+        '0xfac26e3fd40adcdc6652f705d983b4830c00716c',
+        '0x17b4158805772ced11225e77339f90beb5aae968',
+        '0x9c58bacc331c9aa871afd802db6379a98e80cedb',
+        '0xf68fb12aa78e5c2b3eebd71441a990cd5757055c',
+      ],
+    },
+    {
+      input: '0x9c58bacc331c9aa871afd802db6379a98e80cedb',
+      outputs: ['0x775eb53d00dd0acd3ec1696472105d579b9b386b'],
+    },
+    {
+      input: '0xddafbb505ad214d7b80b1f830fccc89b60fb7a83',
+      outputs: ['0x775eb53d00dd0acd3ec1696472105d579b9b386b'],
+    },
+    {
+      input: '0xf68fb12aa78e5c2b3eebd71441a990cd5757055c',
+      outputs: ['0x775eb53d00dd0acd3ec1696472105d579b9b386b'],
+    },
+    {
+      input: '0xfac26e3fd40adcdc6652f705d983b4830c00716c',
+      outputs: ['0x775eb53d00dd0acd3ec1696472105d579b9b386b'],
+    },
+    {
+      input: '0xfadc59d012ba3c110b08a15b7755a5cb7cbe77d7',
+      outputs: ['0x775eb53d00dd0acd3ec1696472105d579b9b386b'],
+    },
+  ],
+};
```

Comment:
I disagree with (1). This file includes utilities for manipulating the large data structure.
I disagree with (2) because such comments would be a lot of work to maintain.
I disagree with (3) because this data object mirrors the response from the price aggregators subgraph so we're relatively certain there won't be typos.